### DISCUSSION TITLE
可以查看gas费的app

### DISCUSSION THREAD

**Discussion by gaoshanliushui662** [rebase-network/hello-world#45] [Ideas]
目前查看gas费非常不方便，想着开发一个app，可以实时查看gas，并做一些监控，想了解下大家有这个需求吗？

**Comment by luckyyang** [comment]
赞。不是有网站可以查嘛？你希望开发手机 app 吗

**Reply by gaoshanliushui662** [reply]
嗯嗯，对，手机app

**Reply by luckyyang** [reply]
感觉监控是比较重要的？在 gas 低的时候做一些合约交互的操作。你能说出几个应用场景吗

**Reply by lw1992** [reply]
我开发了一个苹果app，国内app store不给上架......

**Reply by luckyyang** [reply]
正常，国外 app store 的试了吗

**Reply by lw1992** [reply]
试了，也不行，可能是我的id就是在国内注册的吧

**Reply by gaoshanliushui662** [reply]
1、提供当前实时的gas费及历史gas曲线
2、gas费低于或者高于自己预设的阈值时，提供告警，甚至可以执行一些自动化的脚本命令

**Reply by luckyyang** [reply]
了解了，这个主要是什么样的人会用呢，他们做套利？
